libs/theme/src/ztx.ts

```diff
@@ -62,6 +61,12 @@ export const ztxTheme: CustomTheme = {
     accountName: "Obi Smart Account",
   },
   welcome: {
-    backgroundImage: "/ztx-home.png",
+    background: {
+      backgroundImage: "/ztx-home.png",
+      backgroundColor: "#16151D",
+      backgroundBlendMode: "luminosity",
+      backgroundSize: "cover",
+      backgroundPosition: "center",
```

Comment:
Please remove the `background` prefix from these properties since they are already subproperties of `background`.